### ISSUE TITLE
instructions for indexing database files?

### ISSUE THREAD

**Issue by kapsakcj** [imanyass/ShigaPass#1]
Hi, I was hoping to test out ShigaPass and when I went to run the script, it threw an error that no index files were found.

```
### ipaH checkpoint ###
BLAST Database error: No alias or index file found for nucleotide database [SCRIPT/ShigaPass_DataBases//IPAH/ipaH_150-mers.fasta] in search path [/home/curtis_kapsak/github/ShigaPass::]
```

Could you provide instructions on how to index the databases? I assume I can use the `makeblastdb` command, but wasn't sure what options should be used, if any.

It would be helpful to have these instructions on the main README.md.

Thank you

**Comment by kapsakcj** [comment]
I ended up running `makeblastdb -dbtype nucl <fasta>` on all the fasta files included w/ ShigaPass, and ShigaPass ran successfully afterwards

**Comment by kapsakcj** [comment]
Apologies, I did not see the command line option `-u` for indexing the databases, though it seems that I got the command correct:

https://github.com/imanyass/ShigaPass/blob/1f72e535359adb6f25d47b005e9dc5b042e038c1/SCRIPT/ShigaPass.sh#L112

Maybe it would be worth highlighting on the README.md here, so that users like myself do not miss it! https://github.com/imanyass/ShigaPass#installation

**Comment by tseemann** [comment]
I agree with @kapsakcj 

it seems i can NOT just run `ShigaPass.sh -u` to index the databases?

I have to run a fake full command with `-u` to make it happen?

Maybe add an installation step that runs `-u` on the `EXAMPLE` data?

**Comment by imanyass** [comment]
Yes, that’s correct . The update step needs to be run together with a full command. I’ve updated the GitHub README to make this explicit, so it should be clearer for new users. Thanks for pointing it out!

**Comment by kapsakcj** [comment]
great, thank you for adding that to the README!
